### PR TITLE
Add libvpx to ffmpeg

### PR DESCRIPTION
**Pull request by gpotter2**
- add `libvpx` build for ffmpeg

### CODE REVIEW

File: var/spack/repos/builtin/packages/libvpx/package.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    version('1.10.0', sha256='85803ccbdbdd7a3b03d930187cb055f1353596969c1f92ebec2db839fa4f834a')
+
+    variant('pic', default=True,
+            description='Produce position-independent code (for shared libs)')
+
+    depends_on('yasm')
+
+    def configure_args(self):
+        extra_args = ['--enable-pic']
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/libvpx/package.py]: Shouldn't this depend on whether the `pic` variant is enabled or not?

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/libvpx/package.py (hunk 1 above)
Comment (gpotter2) [reply var/spack/repos/builtin/packages/libvpx/package.py]: Of course. Should be good

**Review by adamjstewart — APPROVED** [review]
Just waiting for @xjrc to review

**Comment by xjrc** [comment]
@gpotter2: Is this variant supported in all versions of `ffmpeg`? If not, please add a `conflicts` statement to constrain this variant to a valid version, and move `libvpx` to a `spec.satisfies` statement at the bottom of the `configure_args` function.

**Comment by gpotter2** [comment]
This is supported since 0.8 (https://github.com/FFmpeg/FFmpeg/blob/82123e133db6d556f3366a1cbb4f0439d70539d4/Changelog#L1268) which is older than the oldest version supported by spack.

**Review by xjrc — APPROVED** [review]

**Pull request merged**